conf/machine/include/rpi-default-providers.inc

```diff
@@ -7,6 +7,8 @@ PREFERRED_PROVIDER_virtual/libgles2 ?= "${@bb.utils.contains("MACHINE_FEATURES",
 PREFERRED_PROVIDER_virtual/libgl ?= "${@bb.utils.contains("MACHINE_FEATURES", "vc4graphics", "mesa", "mesa-gl", d)}"
```

Comment:
I would avoid overcomplicating the git subject. Just avoid subject scope when you touch multiple things: `Add support for optional selection of vlc and ffmpeg`.